manual.py

```diff
@@ -0,0 +1,568 @@
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("RECOMMENDATION from Director Houston Public Works for approval of final contract amount of $2,979,476.73 and acceptance of work on contract with GRAVA LLC for FY2019 Open Drainage System Maintenance Work Orders #1 - 0.68% under the original Contract Amount - DISTRICTS A - PECK; B - JACKSON; C - KAMIN; D - EVANS-SHABAZZ; E - MARTIN; G - HUFFMAN; H - CISNEROS; I - GALLEGOS; J - POLLARD and K - CASTEX-TATUM")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("PROPERTY - NUMBER 3")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("PURCHASING and TABULATION OF BIDS - NUMBERS 4 through 9")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("ORDINANCES - NUMBERS 10 through 26"),
+            matter = #How to add more matters
+                ingestion_models.Matter(
+                "ORDINANCE approving and authorizing third amendment to contract between City of Houston and BRENTWOOD ECONOMIC COMMUNITY DEVELOPMENT CORPORATION, d/b/a BRENTWOOD COMMUNITY FOUNDATION, to extend term of contract and provide additional Housing Opportunities for Persons With AIDS Funds for the continuing administration and operation of a Community Residence and an Emergency Rental Assistance Program with Supportive Services - $892,634.00 - Grant Fund - DISTRICT K - CASTEX-TATUM", 
+                matter_type = "ORDINANCES - NUMBERS 10 through 26",
+                # title??
+                title = "ORDINANCE approving and authorizing Subrecipient Agreement between City of Houston and CATHOLIC CHARITIES OF THE ARCHDIOCESE OF GALVESTON-HOUSTON to provide Emergency Solutions Grant (ESG), and Emergency Solutions Grant Coronavirus (ESG-CV) Funds for administration and related services in connection with the City’s Emergency Rental Assistance Program, for households that have been affected by COVID-19 - $3,058,423.79 - Grant Fund"
+                ),
+            decision = "ADOPTED",
+            votes = [
+            ingestion_models.Vote(
+                person = ingestion_models.Person(
+                    "Amy Peck",
+                    is_active = "true",
+                    # only has seat/role, what to do when only has role??
+                    seat = ingestion_models.Seat(
+                        "District A"
+                    )
+                ),
+                decision = "AYES"
```

Comment:
Where did you find these? I can't find them.

Reply:
I just realized I forgot to leave you comments about checking where to find the votes info. In terms of the votes info I included in the manual.py you asked about in the comment, I found it in the legal minutes on this website https://houston.novusagenda.com/agendapublic/, which states that all voting aye, so I just put down all the members listed.